L1Trigger/TrackFindingTracklet/test/AnalyzerDR.cc

```diff
@@ -186,100 +172,77 @@ namespace trklet {
     set<TPPtr> tpPtrs;
     set<TPPtr> tpPtrsSelection;
     set<TPPtr> tpPtrsPerfect;
-    set<TPPtr> tpPtrsLost;
     int allMatched(0);
     int allTracks(0);
     for (int region = 0; region < setup_->numRegions(); region++) {
-      const int offset = region * channelAssignment_->numNodesDR();
       int nStubs(0);
       int nTracks(0);
-      int nLost(0);
-      for (int channel = 0; channel < channelAssignment_->numNodesDR(); channel++) {
-        vector<vector<TTStubRef>> tracks;
-        formTracks(acceptedTracks, acceptedStubs, tracks, offset + channel);
-        vector<vector<TTStubRef>> lost;
-        formTracks(lostTracks, lostStubs, lost, offset + channel);
-        nTracks += tracks.size();
-        nStubs += accumulate(tracks.begin(), tracks.end(), 0, [](int sum, const vector<TTStubRef>& track) {
-          return sum + static_cast<int>(track.size());
-        });
-        nLost += lost.size();
-        allTracks += tracks.size();
-        if (!useMCTruth_)
-          continue;
-        int tmp(0);
-        associate(tracks, selection, tpPtrsSelection, tmp);
-        associate(tracks, selection, tpPtrsPerfect, tmp, true);
-        associate(lost, selection, tpPtrsLost, tmp);
-        associate(tracks, reconstructable, tpPtrs, allMatched);
-        const StreamTrack& stream = acceptedTracks[offset + channel];
-        const auto end =
-            find_if(stream.rbegin(), stream.rend(), [](const FrameTrack& frame) { return frame.first.isNonnull(); });
-        const int size = distance(stream.begin(), end.base()) - 1;
-        hisChannel_->Fill(size);
-        profChannel_->Fill(channel, size);
-      }
+      vector<vector<TTStubRef>> tracks;
+      formTracks(streamsTrack, streamsStub, tracks, region);
+      nTracks += tracks.size();
+      nStubs += accumulate(tracks.begin(), tracks.end(), 0, [](int sum, const vector<TTStubRef>& track) {
+        return sum += (int)track.size();
```

Comment:
This line was changed to
```
return sum + static_cast<int>(track.size());
```
in #43601, the C style cast and the `+=` are regressions from the preferred form.  (And similar for other instances.)

Reply:
Must have crept in resolving merge conflicts. I cleaned all my accumulate calls.